hw4/tests/task1_test.go

```diff
@@ -0,0 +1,25 @@
+package tests
+
+import (
+	"testing"
+	"time"
+
+	"github.com/COOLizh/epam/epamGolangTasks/hw4/tasks"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestLess(t *testing.T) {
+	assert := assert.New(t)
```

Comment:
You shouldn't use variable name same with any pkg name, it's ambiguous which one is used.